packages/angular_devkit/build_angular/src/webpack/plugins/javascript-optimizer-worker.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import remapping from '@ampproject/remapping';
+import { transform } from 'esbuild';
+import { minify } from 'terser';
+
+/**
+ * A request to optimize JavaScript using the supplied options.
+ */
+interface OptimizeRequest {
+  /**
+   * The options to use when optimizing.
+   */
+  options: {
+    advanced: boolean;
+    define?: Record<string, string>;
+    keepNames: boolean;
+    removeLicenses: boolean;
+    sourcemap: boolean;
+    target: 5 | 2015 | 2017;
```

Comment:
NIT, because we do use 2019 for server builds.

```suggestion
    target: 5 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020;
```